frontend/src/metabase/ui/components/inputs/Select/SelectItem/SelectItem.tsx

```diff
@@ -44,11 +36,6 @@ export const SelectItem = forwardRef(function SelectItem(
       flex={1}
       aria-selected={selected}
       {...props}
-    >
-      {icon && <Icon name={icon} />}
-      <Text c="inherit" lh="inherit">
-        {label}
-      </Text>
-    </Group>
+    />
```

Comment:
For the purpose of full customization, `SelectItem` is now the one where you need to provide `children`.
`DefaultSelectItem` is the one that accepts `icon` and `label` props.